### DISCUSSION TITLE
section8: FundMe Frontend Error

### DISCUSSION THREAD

**Discussion by wilson0922** [Cyfrin/foundry-full-course-cu#2060] [Q&A] [answered]
Hi there,

I followed the instruction and successfully connected the contract. However, when I started to fund ether, I got error as below, does someone know how to solve it? Thanks!

![Untitled](https://github.com/user-attachments/assets/f4bb821b-541c-4892-8bcb-fd436d6b35f1)

**Comment by EngrPips** [comment, marked as answer]
Hello @wilson0922, Did you send the amount that meets the minimum requirement of the contract?

**Reply by wilson0922** [reply]
> Hello @wilson0922, Did you send the amount that meets the minimum requirement of the contract?

Hi @EngrPips You saved me! It reminded me when I set anvil network, the suggested currency uint was go, so it didn't meet the  minimum requirement. Thank you so much!

![Untitled](https://github.com/user-attachments/assets/86408616-a423-435d-8f60-46f30cb927c1)

**Reply by EngrPips** [reply]
You are welcome, Friend. Keep crushing it.
